Automattic/wp-calypso
# Paid Newsletter: Update status indicator to show progress in Stepper

This PR adds updates the status indicator for the Stepper of the paid-newsletter flow. 

Also needs D159083-code to test the flow. 

## Why are these changes being made?
* So that we can visually indatce to the user what the state of the importer is as they progress though the rest of the flow. 

## Testing Instructions

Apply D159083 if it hasn't landed yet to your sandbox. 
And map public-api.wordpress.com 
then start the async job watcher. 
Start the content importer process. 

Notice that the indicator changes as the request for the paid-importer endpoint are made. 


## Pre-merge Checklist
